vision/transforms/transforms.go

```diff
@@ -13,3 +18,52 @@ type NormalizeTransformer struct {
 func Normalize(mean float64, stddev float64) *NormalizeTransformer {
 	return &NormalizeTransformer{mean, stddev}
 }
+
+// ComposeTransforms composes transforms together
+type ComposeTransforms struct {
```

Comment:
Compose is a **transformer**, so the name should be `ComposeTransformer`?

Reply:
Agree with that, will rename it in the next PR.